assets/app.js

```diff
@@ -266,7 +266,33 @@ new Vue({
                 ]
             }
         },
-        translationUpdates: LogData.content
+        translationUpdates: LogData.content,
+        resourceDict: {
+            baseProducts: {
+                list: this.baseProducts,
+                name: "Base Products",
+                selectableOptions: "optionsBaseProduct",
```

Comment:
use `this.optionsBaseProduct`

Reply:
I cannot use compute in data because I found that the data is evaluated before the computed array, so `this.optionsBaseProduct`  doesn't exist yet

and I wrote wrong in    list: this.baseProducts I will change to list: "baseProducts "